docs/installation/binary_installation.md

```diff
@@ -0,0 +1,58 @@
+# Binary Installation
+
+Visualization packages with CARET can also be installed in binary.
+
+<prettier-ignore-start>
+!!! Note
+    Only the visualization packages can be installed by binary installation.
+    The binary does not include measurement packages.
+<prettier-ignore-end>
+
+CARET is confirmed to run on the platforms shown in the following table with supported version.
+
+| dependent platform | supported version |
+| ------------------ | ----------------- |
+| ROS                | Humble            |
+| Ubuntu             | 22.04             |
+| LTTng              | stable-2.13       |
+| Linux Kernel       | 5.15.x            |
+| Python3            | 3.10.x            |
+
+## Installation of required packages
+
+Update your apt repository caches.
+
+```
+sudo apt update
+```
+
+The following packages are required for the CARET binary packages. These dependencies are not automatically installed with the binary installation.
+
+```
+sudo apt install python3-bt2
+python3 -m pip install -U \
+  'pandas>=1.4.0' \
+  bokeh \
+  jupyterlab \
+  multimethod
+```
```

Comment:
The version specified here seems to be different from the version in [requirements.txt](https://github.com/tier4/caret_analyze/blob/main/src/requirements.txt).
Without a valid reason, we would like to follow the version in requirements.txt.

Reply:
I saw old documents.

Reply:
Fixed in https://github.com/tier4/caret_doc/pull/335/commits/4fade85ffe013f251ac6b9295a838ca0a8821371.